tombstone_gc.cc

```diff
@@ -217,6 +219,29 @@ void tombstone_gc_state::update_repair_time(table_id id, const dht::token_range&
     *m += std::make_pair(locator::token_metadata::range_to_interval(range), repair_time);
 }
 
+void tombstone_gc_state::insert_pending_repair_time_update(table_id id,
+        const dht::token_range& range, gc_clock::time_point repair_time, shard_id shard) {
+    _pending_updates[id].push_back(range_repair_time{range, repair_time, shard});
+}
+
+future<> tombstone_gc_state::flush_pending_repair_time_update(replica::database& db) {
+    auto pending_updates = std::move(_pending_updates);
```

Comment:
`std::exchange(_pending_updates, {})`

Reply:
OK.

Reply:
Why?

Reply:
Won't `tombstone_gc_state::_pending_updates` be ever reused?

Reply:
We will reuse the _pending_updates. The moved-from objects shall be placed in a valid but unspecified state. So it is better to use std::exchange here to make it an empty container.